### ISSUE TITLE
Lower the Gaslimit

### ISSUE THREAD

**Issue by filipe-te** [HashLips/hashlips_nft_minting_dapp#99]
Hey Guys, 

i made a fun Project and set the mint price to 0.001 eth… Unfortunately the Gas fees are so high that nobody (me included:D) wants to mint. Gas is around $25-$50 per Mint. I tried to lower the Gas limit but then minting failed. 

Is there a way to lower the Gas fees?

**Comment by ferrerjavi** [comment]
you have to optimise your smart contract :)

**Comment by filipe-te** [comment]
ok thanks :) Any idea how I can do that?^^
I already deployed it to ETH Mainnet...

**Comment by ferrerjavi** [comment]
Yeah sure, anyway if it's only 25-30$ extra per transaction I see it low for ETH, the average gas is 40$ right now.... There is nothing you can do there
